src/sanitize/sanitizeNumbers.spec.ts

```diff
@@ -1157,4 +1161,76 @@ describe('sanitizeNumbers', () => {
 			'5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY',
 		]);
 	});
+
+	describe('handles metadata v14 correctly', () => {
+		const sanitizeOptions = {
+			isMetadata: true,
+			registry: polkadotRegistryV9190,
+		};
+
+		it('handles unknown struct bytes correctly for u128', () => {
+			const struct = new Struct(
+				polkadotRegistryV9190,
+				{ type: 'Text', value: 'Bytes' },
+				{ type: '6', value: '0x00e87648170000000000000000000000' }
```

Comment:
Is type 6 a sequencew of `u8`?

Reply:
ah or is it more like `[u8; 16]`?

Reply:
So type 6 for polkadot as defined is:

```
{
  "id": "6",
    "type": {
    "path": [],
      "params": [],
        "def": {
      "primitive": "U128"
    },
    "docs": []
  }
},
```

Reply:
Yea I can add comments, it might be a little redundant because the title of each test refers to what its looking for, but it doesnt hurt to be more explicit.